src/webview.rs

```diff
@@ -114,35 +117,50 @@ impl Window {
                 }
             }
             e => {
-                log::warn!("Verso WebView isn't supporting this message yet: {e:?}")
+                log::trace!("Verso WebView isn't supporting this message yet: {e:?}")
             }
         }
     }
 
-    /// Handle servo messages with main panel.
+    /// Handle servo messages with main panel. Return true it requests a new window.
     pub fn handle_servo_messages_with_panel(
         &mut self,
         panel_id: WebViewId,
         message: EmbedderMsg,
         sender: &Sender<ConstellationMsg>,
         clipboard: Option<&mut Clipboard>,
-    ) {
+        compositor: &mut IOCompositor,
+    ) -> bool {
```

Comment:
jus wondering, maybe should return Result instead of bool?

Reply:
Result means something might get wrong, but this bool is about whether it should request new window.
If we have an error message need to return. I'll update it to Result type.